app/Livewire/Pages/Store.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Livewire\Pages;
+
+use App\Models\Product;
+use Illuminate\Pagination\LengthAwarePaginator;
+use Illuminate\Support\Collection;
+use Illuminate\View\View;
+use Livewire\Component;
+use Livewire\WithPagination;
+use Shopper\Core\Models\Attribute;
+
+class Store extends Component
+{
+    use WithPagination;
+
+    public Collection $atributes;
+
+    /** @var string[] */
+    public array $selectedAttributes = [];
+
+    public function mount(): void
+    {
+        $this->atributes = Attribute::with('values')->where('is_enabled', true)->get();
```

Comment:
```suggestion
        $this->atributes = Attribute::with('values')->enabled()->get();
```
Le scope enabled est deja defini dans le model attribute

Reply:
c'est fait